### DISCUSSION TITLE
Printer not connected

### DISCUSSION THREAD

**Discussion by onil80** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2579] [Q&A]
suddenly, after changing a fan to my Ender 3 .... (I state I have a motherboard skr 1.3 and a tft35 touch screen) the printer no connected to the display .... do you have any ideas?

**Comment by kisslorand** [comment]
It's not FW related problem for sure.

**Comment by onil80** [comment]
I checked all the links ....

**Comment by kisslorand** [comment]
I checked the FW, it will not stop communication "after changing a fan".

**Comment by onil80** [comment]
i know .... but what should i check?

**Comment by kisslorand** [comment]
I cannot suggest anything than the cables and/or the new fan.

**Comment by onil80** [comment]
I found the problem ..... it was the thermistor shorting the motherboard
